changelog.d/20240405_091941_klakhov_rar_support.md

```diff
@@ -0,0 +1,4 @@
+### Added
+
+- Support for `.rar` archives
```

Comment:
Wouldn't this patch also add support for a bunch of other archive types? If I understand correctly, previously only `.zip` was supported.